airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -282,6 +319,11 @@ class Events(SemiIncrementalGithubStream):
         "org",
     )
 
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        link = response.headers.get("Link")
+        if 'rel="next"' in link:
```

Comment:
`link` may be `None`, so an exception will be raised

Reply:
Done.